stage0/src/acpi.rs

```diff
@@ -384,7 +384,7 @@ impl RomfileCommand {
     }
 }
 
-pub fn build_acpi_tables(fwcfg: &mut FwCfg) -> Result<(), &'static str> {
+pub fn build_acpi_tables(fwcfg: &mut FwCfg) -> Result<u64, &'static str> {
```

Comment:
Could you add a comment explaining what is being returned?

Reply:
Done.